CRM/Mailing/Event/BAO/Unsubscribe.php

```diff
@@ -258,12 +258,12 @@ public static function &unsub_from_mailing($job_id, $queue_id, $hash, $return =
     if ($groupIds || $baseGroupIds) {
       $groupIdClause = "AND $group.id IN (" . implode(', ', array_merge($groupIds, $baseGroupIds)) . ")";
     }
-    $do->query("
+    $do = CRM_Core_DAO::executeQuery("
             SELECT      $group.id as group_id,
                         $group.title as title,
                         $group.description as description
-            FROM        $group
-            LEFT JOIN   $gc
+            FROM        civicrm_group $group
```

Comment:
Again why are we specifiying the table name here rather than in $group?

Reply:
@eileenmcnaughton can you comment on this please?

Reply:
right so my understanding . is it was because we weren't relying on l10n in the query object - but @seamuslee001 believes it might not be available via extern path